UPGRADE.md

```diff
@@ -15,6 +15,17 @@ INSERT INTO `re_operators` (`id`, `operator`, `type`, `inputType`) VALUES
     (19, '!=', 6, 'auto-complete');
 ```
 
+Additionally the filter by country has changed. Run following SQL script to update your filter conditions:
+
+```sql
+UPDATE `re_conditions` SET `field` = 'countryId', `type` = 6, `value` = CONCAT('"', (SELECT `id` FROM `co_countries` WHERE `code` = REPLACE(`re_conditions`. `value`, '"', '') LIMIT 1), '"') WHERE `field` = 'countryCode' AND `operator` != 'LIKE';
```

Comment:
the values will be `json_encode` by doctrine. if this is not happen for you something is wrong. are you sure that you are in the newest master branch (not on 1.1.9) when testing?